source/module_hsolver/kernels/rocm/math_kernel_op.hip.cu

```diff
@@ -653,6 +653,26 @@ void axpy_op<std::complex<double>, base_device::DEVICE_GPU>::operator()(const ba
     hipblasErrcheck(hipblasZaxpy(cublas_handle, N, (hipblasDoubleComplex*)alpha, (hipblasDoubleComplex*)X, incX, (hipblasDoubleComplex*)Y, incY));
 }
 
+hipblasOperation_t judge_trans_op(bool is_complex, const char& trans)
+{
+    if (trans == 'N')
+    {
+        return CUBLAS_OP_N;
```

Comment:
`HIPBLAS_OP_N` and same bug in code below.

Reply:
Fixed in commit `e54b2fe`. Also discovered bugs in `gemm_op` and fixed them.